src/js/claims-status/components/appeals-v2/AppealHeader.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+const AppealHeader = ({ heading, lastUpdated }) => {
+  moment.updateLocale('en', {
+    meridiem: (hour) => {
+      if (hour < 12) {
+        return 'a.m.';
+      }
+      return 'p.m.';
+    },
+  });
+
+  const formattedDateTime = moment(lastUpdated).format('MMMM DD, YYYY, [at] h:mm a [(ET)]');
+  const description = `Up to date as of ${formattedDateTime}`;
+
+  return (
+    <div className="appeal-header">
+      <h1>{heading}</h1>
+      <p>{description}</p>
```

Comment:
I thought so as well, though chose not to do anything about it for the time being since this should just use the 'standard' spacing between h1 and other content that we use across the site. Happy to tweak this if that's what you think, Gina!